requirements.in

```diff
@@ -2,10 +2,11 @@
 #
 # Generates requirements.txt file.
 
-boto3==1.28.58
+boto3==1.28.68
+botocore==1.31.68
```

Comment:
I added botocore because it makes it easier to fix the project when it's in a state where we can't update requirements and can't build it. We don't explicitly use botocore in the code.

Part of me thinks we should spend some more time making this project more maintainable, but when we switch to GCP, we'll drop this project altogether for something different that's GCP-centric.